curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -7,7 +7,6 @@ module UploadSetBehavior
     included do
       has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
 
-      property :creator, predicate: ::RDF::DC.creator
```

Comment:
I'm cool with that. I still have a question about wiping out the callback hooks above, which Sufia already uses to send notifications.